docs/articles_en/learn-openvino/llm_inference_guide/genai-guide.rst

```diff
@@ -218,6 +218,114 @@ Specify generation_config to use grouped beam search:
             cout << pipe.generate("The Sun is yellow because", config);
          }
 
+Efficient Text Generation via Speculative Decoding
+##################################################
+
+Speculative decoding (or assisted-generation) enables faster token generation
+when an additional smaller draft model is used alongside the main model.
+The draft model predicts the next K tokens one by one in an autoregressive manner,
+while the main model validates these predictions and corrects them if necessary.
+
+Each predicted token is compared, and when there is a difference between the draft and
+main model, the last token predicted by the main model is kept. Then, the draft
+model acquires this token and retries prediction of the next K tokens,
```

Comment:
Please use "tries" instead of "retries". It is a mistake in the readme, which will be fixed as well

Reply:
I have applied requested changes: https://github.com/openvinotoolkit/openvino/pull/27760/commits/38597da740babf3be6cb1279476831e330d89c0b